Amazon.IonDotnet/Internals/Binary/RawBinaryReader.cs

```diff
@@ -600,7 +600,7 @@ protected decimal ReadDecimal(int length)
 
             ReadVarInt(out var exponent);
             if (exponent > 0)
-                throw new IonException($"Exponent should be <= 0: {exponent}");
+                return 0m;
```

Comment:
Please explain this.

Reply:
for a case like `69 80 0F D0 81 81 80 80 80 81    // The same instant with 0d1 fractional seconds` , previous implementation was throwing exception whereas it is a valid timestamps.

Reply:
So does this path only get hit when the mantissa is 0? That's the part I'm confused by.

Reply:
`_localRemaining` is being used to keep track of mantissa is this method.
I changed the logic so we return 0 if mantissa is 0 and kept the original behaviour for when exponent > 0 and we have a non-zero mantissa.